README.md

```diff
@@ -131,6 +131,48 @@ In general, you can send unlock commands as often and as quickly as you wish, be
 
 Failing to wait on setup can cause the lock to enter a state where it can't be setup until we reset it on the server. We have tried a number of different strategies to ease or prevent this issue, but as yet have not been 100% successful in all cases. Thus, it is in your best interest to specifically wait 60 seconds after the first unlock attempt of a new lock before trying again. This is also true after an old lock is reset (limited keys cleared from the database).
 
+### Updating Lock Firmware
+As with any other software the lock firmware needs updating from time to time. This can include for fixing bugs on the locks itself or for adding new functionality. Unfortunately, updating the lock firmware must be done one-by-one with each lock in-hand. There a number of reasons for this. The major reason is that there needs to be a stable Internet connection for downloading the new firmware directly to the lock.
+
```

Comment:
Look at removing the word "Unfortunately" to avoid negatively coloring the documentation